go.mod

```diff
@@ -1,14 +1,18 @@
-module github.com/UpCloudLtd/upcloud-go-api/v4
+module github.com/UpCloudLtd/upcloud-go-api/v5
 
-go 1.14
+go 1.18
```

Comment:
Thanks @villevsv-upcloud for review! 

I would stay in 1.18 so that we don't accidentally add any 1.19 features and interfere with lib clients that are still in 1.18.